tests/imports/nested_imports.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import torch
+
+import lightly
+
+
+class TestNestedImports(unittest.TestCase):
+
+    def test_nested_imports(self):
+        # active learning (commented out don't work)
+        #lightly.active_learning.agents.agent.ActiveLearningAgent
```

Comment:
Could you please create a follow-up issue for how to improve the handling of the imports? And mention in the issue that some of the direct imports don't work at the moment (the ones commented out here)

Reply:
Yes